main.py

```diff
@@ -3,5 +3,36 @@
 from django.db.models import QuerySet
 
 
+from db.models import Genre
+from db.models import Actor
+
+
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [
+        "George Klooney",
+        "Kianu Reaves",
+        "Scarlett Keegan",
+        "Will Smith",
+        "Jaden Smith",
+        "Scarlett Johansson"
+    ]
+
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    for actor in actors:
+        Actor.objects.create(
+            first_name=actor.split()[0],
```

Comment:
You don't need to split() every time. Just use unpacking.
Here some tips https://github.com/mate-academy/py-genres-and-actors/blob/master/checklist.md